src/constants.js

```diff
@@ -38,6 +38,7 @@ export function getModuleCacheDirectory(): string {
 export const MODULE_CACHE_DIRECTORY = getModuleCacheDirectory();
 export const LINK_REGISTRY_DIRECTORY = `${MODULE_CACHE_DIRECTORY}/.link`;
 export const GLOBAL_MODULE_DIRECTORY = `${MODULE_CACHE_DIRECTORY}/.global`;
+export const CACHE_FILENAME = path.join(userHome, '.yarn', '.roadrunner.json');
```

Comment:
I would like to use this variable in `bin/yarn.js`. But there i cannot simple import this file right? currently this is duplicate here and in `bin/yarn.js`